site/content/docs/snippets/_index.md

```diff
@@ -21,24 +21,43 @@ Snippets can be useful for:
 
 ## How to use snippets
 
-To create a snippet, access <a href="https://app.checklyhq.com/snippets" target="_blank">the snippets section on the left side of the UI</a>. 
+To create a snippet, access <a href="https://app.checklyhq.com/snippets" target="_blank">the snippets section on the left side of the UI</a>.
 
-When creating a snippet, note that the snippet name will be used as its filename. It's not necessary to name the snippet with the `.js` file extension, though, since this will be added automatically.
+When creating a snippet, note that the snippet name will be used as its filename. It's not necessary to name the snippet with the `.js` file extension since this will be added automatically.
 
-The snippet can then be imported in Browser checks as well as setup and teardown scripts using the Node.js `require` function. When a check is executed, snippets will be available in the `./snippets` directory. Snippets work like any Javascript file on your local disk in Node.js, making it possible to expose functions and properties on the `module.exports` object.
+A snippet can be imported in Browser checks as well as setup and teardown scripts using the Node.js `require` function. When a check or script runs, snippets will be available in the `./snippets` directory.
 
-To require a snippet named `setup-library`, a check would use:
+A simplified file and directory structure for API and Browser checks looks as follows:
+
+```
+├── browser-check.js
```

Comment:
The file will be run as  `script.js`. This probably doesn't matter for 99% of checks, but users can see it with `__filename`.

Reply:
Uff... Tricky... :D 

Okay let me adjust. :D